lib/smartcar.rb

```diff
@@ -94,14 +99,18 @@ def get_compatibility(vin:, scope:, country: 'US', options: {})
     #
     # API Documentation - https://smartcar.com/docs/api#get-user
     # @param token [String] Access token
+    # @param version [String] Optional API version to use, defaults to what is globally set
+    # @param options [Hash] Other optional parameters including overrides
+    # @option options [Faraday::Connection] :service Optional connection object to be used for requests
     #
     # @return [OpenStruct] And object representing the JSON response mentioned in https://smartcar.com/docs/api#get-user
     #  and a meta attribute with the relevant items from response headers.
-    def get_user(token:, version: Smartcar.get_api_version)
+    def get_user(token:, version: Smartcar.get_api_version, options: {})
```

Comment:
The `get_user` and `get_vehicles` method signatures are a _little_ inconsistent with `get_compatibility` now since in some cases `version` is supplied in options and other times it's a top level attribute. I considered accepting it inside of `options` in these two methods as well, but figured it would be better to leave that up to you all so you can decide how you want to handle deprecating it at the top level.